packages/landing/constants/index.ts

```diff
@@ -0,0 +1,4 @@
+export const STACKLY_LANDING_URL =
+  process.env.STACKLY_LANDING_URL ?? "https://stackly.app/";
+export const STACKLY_APP_URL =
+  process.env.STACKLY_APP_URL ?? "https://stackly-ui.pages.dev/"; // "https://stackly.eth.limo/";
```

Comment:
No need for the comment here. And we can point to the correct default URL. `STACKLY_APP_URL` can point to `https://stackly-ui.pages.dev/`.

Reply:
And what about updating the default value? `"https://stackly-ui.pages.dev/"` to `"https://stackly.eth.limo/"`

Reply:
fixed.